src/key-value.js

```diff
@@ -10,19 +10,29 @@ export default React.createClass({
       valueItem: PropTypes.string
     })),
     onChange: PropTypes.func,
-    customAddButtonRenderer: PropTypes.func
+    customAddButtonRenderer: PropTypes.func,
+    keyInputPlaceholder: PropTypes.string,
+    valueInputPlaceholder: PropTypes.string,
+    hideKeyInputLabel: PropTypes.bool,
+    hideValueInputLabel: PropTypes.bool
   },
   getDefaultProps() {
     return {
       rows: [],
-      onChange: () => {}
+      onChange: () => {},
+      keyInputPlaceholder: 'Enter the meta key\'s title',
+      valueInputPlaceholder: 'Enter the value',
+      hideKeyInputLabel: false,
+      hideValueInputLabel: false
     };
   },
   getInitialState() {
     return {
       rows: [
         ...this.props.rows
-      ]
+      ],
+      hideKeyInputLabel: this.props.hideLabels,
```

Comment:
It's not necessary to put `hideLabels` in state (it doesn't change within the component). Btw, there is no `hideLabels` in props but there're `hideKeyInputLabel` and `hideValueInputLabel`.